clients.go

```diff
@@ -216,3 +239,40 @@ func headersToHTTPHeaders(h *headersList) http.Header {
 	}
 	return headers
 }
+
+func executeLuaScript(l *lua.LState, path string, method string, body string) (string, string, string, error) {
+
+	if err := l.CallByParam(lua.P{
+		Fn:      l.GetGlobal("request"), // name of Lua function
+		NRet:    4,                      // number of returned values
+		Protect: true,                   // return err or panic
+	}, lua.LString(path), lua.LString(method), lua.LString(body)); err != nil {
+		return path, method, body, err
+	}
+
+	var err error
+
+	if str, ok := l.Get(-1).(lua.LString); ok {
+		err = fmt.Errorf(str.String())
+	}
+	l.Pop(1)
+
+	if str, ok := l.Get(-1).(lua.LString); ok {
+		body = str.String()
+	}
+	l.Pop(1)
+
+	if str, ok := l.Get(-1).(lua.LString); ok {
+		method = str.String()
```

Comment:
What if user's script returns something like "ABRACADABRA" or "AVADA KEDABRA!"? `fasthttp` in particular won't be very happy with this :smile: